crates/op-rbuilder/Cargo.toml

```diff
@@ -97,7 +97,7 @@ serde_yaml = { version = "0.9" }
 
 
 # `flashblocks` branch
-rollup-boost = { git = "http://github.com/flashbots/rollup-boost", rev = "60885346d4cf7f241de82790478195747433d472" }
+rollup-boost = { git = "http://github.com/flashbots/rollup-boost", rev = "6f994a2092967108e7dda0ade57625ba10940310" }
```

Comment:
What did you change here?

Reply:
https://github.com/flashbots/rollup-boost/pull/234